src/Templates/src/templates/maui-blazor/Shared/SurveyPrompt.razor

```diff
@@ -1,16 +1,16 @@
 ﻿<div class="alert alert-secondary mt-4">
-	<span class="oi oi-pencil me-2" aria-hidden="true"></span>
-	<strong>@Title</strong>
+    <span class="oi oi-pencil me-2" aria-hidden="true"></span>
+    <strong>@Title</strong>
 
-	<span class="text-nowrap">
-		Please take our
-		<a target="_blank" class="font-weight-bold" href="https://go.microsoft.com/fwlink/?linkid=2149017">brief survey</a>
-	</span>
-	and tell us what you think.
+    <span class="text-nowrap">
+        Please take our
+        <a target="_blank" class="font-weight-bold link-dark" href="https://go.microsoft.com/fwlink/?linkid=2148851">brief survey</a>
+    </span>
+    and tell us what you think.
 </div>
 
 @code {
-	// Demonstrates how a parent component can supply parameters
-	[Parameter]
-	public string Title { get; set; }
+    // Demonstrates how a parent component can supply parameters
+    [Parameter]
+    public string? Title { get; set; }
```

Comment:
And to confirm, this doesn't generate any warnings regarding nullability in the template with all the default settings?

Reply:
Maybe, let me check.

We might want to enable nullable on these templates to match the rest of the templates, wouldn't we?

Reply:
Eventually I think all templates should have nullable enabled. Not sure where MAUI is at yet.

Reply:
Core is nullable, but Controls is not. We probably should start on that - even if it is just 1 file at a time and add `#nullable` as we go. This could also be a light entry point for contributions.